assets/views/home/editProfile.ejs

```diff
@@ -14,7 +14,8 @@
               <label class="col-sm-3 control-label">Introduction:</label>
 
               <div class="col-md-8">
-                <textarea type="text" class="form-control" ng-model="user.intro"></textarea>
+                <textarea type="text" class="form-control" ng-model="user.intro" ng-maxlength="10000" maxlength="10000"></textarea>
+                <span style="float:right">{{user.intro.length}}/10000</span>
```

Comment:
👌

Reply:
README reverted

Reply:
The way this is now, there's no number displayed before you enter text, which looks a bit awkward (https://i.imgur.com/9avLQEO.png). Could we fix this by changing `{{user.intro.length}}` to `{{user.intro.length || 0}}`?